lib/raft/raft_test.cpp

```
@@ -0,0 +1,184 @@
+        RequestVoteResponse &failedResponse = rvResponses.emplace_back();
+        failedResponse.set_votegranted(0);
+        failedResponse.set_responderid(id);
+        failedResponse.set_term(1);
+
+        RequestVoteResponse &response = rvResponses.emplace_back();
+        response.set_votegranted(1);
+        response.set_responderid(id);
+        response.set_term(2);
+        EXPECT_CALL(*m_raftStubs[id], RequestVote)
+            .Times(testing::Exactly(2))
+            .WillOnce(testing::DoAll(testing::SetArgPointee<2>(failedResponse), testing::Return(grpc::Status::OK)))
```

Cppcheck (2.10-2)

[error] Reference to rvResponses that may be invalid.

(invalidContainerReference)